File: Assignment_05/question_1.c
//////////////////////////////////////////////////////////////////////////////////////////// 
// Name: Pattern
// Description: To print $ and * on screen (number from input)
// Input: void
// Output: void
// Author: Ritesh Jillewad
// Date: 14-05-2025
////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

void Pattern(int iNo)
{
    int iCnt = 0;

    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf(" $ * ",iCnt);
    }
}

int main()
{
    int iValue;

    printf("Enter a Number:\n");
    scanf("%d",&iValue);

    Pattern(iValue);

    return 0;
}

File: Assignment-08/question_1.java
///////////////////////////////////////////////////////////////////////////
/// Name: CirleArea
/// Description: Accepts radius and calculate it's area
/// Input: float
/// Output: float
/// Date: 19-05-2025
/// Author: Ritesh Jillewad
///////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Circle
{
    public double CircleArea(double fRadius)
    {
        final double PI = 3.1;
        double fArea = (PI * fRadius * fRadius);
        return fArea;
    }
}

class question_1
{
    public static void main(String R[])
    {
       Scanner sobj = new Scanner(System.in);
       double dValue = 0.0;
       double dRet = 0.0;
       
       System.out.println("Enter Radius of Circle:");
       dValue = sobj.nextDouble();
       
       Circle cobj = new Circle();
       dRet = cobj.CircleArea(dValue);
       System.out.println("The Area of Circle is: " + dRet);

       sobj.close();
    }
}